data/helpers.d/backend

```diff
@@ -151,6 +159,18 @@ ynh_add_nginx_config () {
 	if test -n "${final_path:-}"; then
 		ynh_replace_string "__FINALPATH__" "$final_path" "$finalnginxconf"
 	fi
+
+	# Replace all other variable given as arguments
+	for v in $others_var
+	do
+        ynh_replace_string "__${v^^}__" "${!v}" "$finalnginxconf"
+	done
+	
+	if [ "${path_url:-}" != "/" ]
+	then
+		ynh_replace_string "^#sub_path_only" "" "$finalnginxconf"
+	fi
```

Comment:
Good idea to manage that here, that has to be done.
But, as we doing that here, we should also manage `#root_path_only`

Reply:
Should be also fixed